src/coreclr/vm/comcallablewrapper.cpp

```diff
@@ -3851,6 +3851,7 @@ void ComMethodTable::SetITypeInfo(ITypeInfo *pNew)
     }
     CONTRACTL_END;
 
+    ExecutableWriterHolder<ComMethodTable> comMTWriterHolder(this, sizeof(ComMethodTable));
```

Comment:
This helper won't flip the permission on the address. Instead, it is going to create a second writeable mapping of the page somewhere else.  You can get to that address by calling `comMTWriterHolder.GetRW()`.

Reply:
Thanks. I should have paid more attention to where I was looking for usage examples.

https://github.com/dotnet/runtime/blob/8582c5cbcf03d6c0d1d3f0d11e622b8b0168f50f/src/coreclr/vm/comcallablewrapper.h#L415-L421